ictv/templates/screens.html

```diff
@@ -1,155 +1,164 @@
-            $message
+            <h4><i class="icon fa {{icon}}"></i>{{title}}</h4>
+            {{message}}
         </div>
+    {%- endmacro %}
 
-    $if feedbacks.has('create', 'ok'):
-        $:create_alert("Added", "This screen has been successfully added.", "fa-check")
-
-    $if feedbacks.has('edit', 'ok'):
-        $:create_alert("Edited", "This screen has been successfully edited.", "fa-check")
+    {% if feedbacks.has('create', 'ok') %}
+        {{ create_alert("Added", "This screen has been successfully added.", "fa-check") | safe }}
+    {% endif %}
 
-    $if feedbacks.has('delete', 'ok'):
-        $:create_alert("Deleted", "This screen has been successfully deleted.", "fa-trash")
+    {% if feedbacks.has('edit', 'ok') %}
+        {{ create_alert("Edited", "This screen has been successfully edited.", "fa-check") | safe }}
+    {% endif %}
 
-    $if feedbacks.has('subscribe', 'ok'):
-        $:create_alert("Applied", "These changes have been successfully applied.", "fa-rss")
+    {% if feedbacks.has('delete', 'ok') %}
+        {{ create_alert("Deleted", "This screen has been successfully deleted.", "fa-trash") | safe }}
+    {% endif %}
 
-    $if feedbacks.has('chown', 'ok'):
-        $:create_alert("Applied", "These changes have been successfully applied.", "fa-user")
+    {% if feedbacks.has('subscribe', 'ok') %}
+        {{ create_alert("Applied", "These changes have been successfully applied.", "fa-rss") | safe }}
+    {% endif %}
 
-    $if feedbacks.has('configure', 'ok'):
-        $:create_alert("Configured", "This screen has been successfully configured.", "fa-wrench")
+    {% if feedbacks.has('chown', 'ok') %}
+        {{ create_alert("Applied", "These changes have been successfully applied.", "fa-user") | safe }}
+    {% endif %}
 
+    {% if feedbacks.has('configure', 'ok') %}
+        {{ create_alert("Configured", "This screen has been successfully configured.", "fa-wrench") | safe }}
+    {% endif %}
     <div class="box">
         <div class="box-body">
             <table id="table" class="table table-hover table-bordered" cellspacing="0" width="100%" data-page-length='25'>
                 <thead>
                     <tr>
-                        <th $:make_tooltip(info['screen']['name'])>Name</th>
-                        <th $:make_tooltip(info['screen']['building'])>Building</th>
-                        <th $:make_tooltip(info['screen']['location'])>Location</th>
+                        <th {{make_tooltip(info['screen']['name']) | safe }}>Name</th>
+                        <th {{make_tooltip(info['screen']['building']) | safe }}>Building</th>
+                        <th {{make_tooltip(info['screen']['location']) | safe }}>Location</th>
                         <th>Orientation</th>
-                        $if UserPermissions.administrator in user.highest_permission_level:
+                        {% if UserPermissions.administrator in highest_permission_level %}
```

Comment:
May I probably have missed where ``user.`` has gone and if `` highest_permission_level`` has become a template argument ?

Reply:
```highest_permission_level``` is indeed passed as a template argument to prevent ```dict object has no attribute highest_permission_level``` from popping. 
In some way the ORM magic method ```__getattr__``` is not triggered during the template rendering as it was previously with templator.

Reply:
If it's actually the case, it would be worth to investigate the exact reason to avoid any similar issue in the code.

Reply:
A wild guess would be that jinja2 uses the __dict__ representation of the objects and this would explain it pretty simply. I'll try to confirm this.

Reply:
This makes sense, maybe it's worth adding a comment in the code to remember.